controllers/controller.js

```diff
@@ -30,9 +30,16 @@ exports.getReviews = (req, res, next) => {
 
 exports.getReviewsById = (req, res, next) => {
   const { review_id } = req.params;
-  fetchReviewById(review_id)
+  fetchReviews()
     .then((result) => {
-      res.send(result);
+      result.forEach((review) => {
+        if (review.review_id === parseInt(review_id)) {
+          res.send(review);
+        }
+      });
+      if (parseInt(review_id) == review_id) {
+        return Promise.reject({ status: 404, msg: "id not found" });
+      } else return Promise.reject({ status: 400, msg: "bad request" });
```

Comment:
why has this code been changed for this ticket? This ticket is for the individual review, not all reviews.

Reply:
I've used fetchReviews in the model to return back an array of all the reviews to the controller (including the comment_count) and then used the review_id parameter to pick out the single review from this array to then be sent.